save3ds_fuse/Cargo.toml

```diff
@@ -15,5 +15,5 @@ stderrlog = "0.5"
 
 [target.'cfg(unix)'.dependencies]
 libc = { version = "0.2", optional = true }
-fuse = { version = "0.3.1", optional = true }
+fuser = { version = "0.12.0", optional = true }
 time = { version = "0.1.42", optional = true }
```

Comment:
This crate was added because the old `fuse` used it. Does the new `fuser` still use it (this particular old version), or we can remove/update it?